resources/object_repo/import_dbaasinventory_obj.resource

```diff
@@ -21,7 +21,8 @@ ${txtBoxRdsAwsSecKeyID}         //input[@id='rds-registration-AWS_SECRET_ACCESS_
 ${txtBoxRdsAwsRegion}           //input[@id='rds-registration-AWS_REGION']
 ${txtBoxAckResrceTags}          //input[@id='rds-registration-ACK_RESOURCE_TAGS']
 ${txtBoxAckLogLvl}              //input[@id='rds-registration-ACK_LOG_LEVEL']
-${btnImportEnabled}             //button[contains(.,"Import") and @aria-disabled="false"]
+${btnImportEnabled}             //button[@class="pf-c-button pf-m-primary submit-button"]
```

Comment:
@ikanias IMHO, we can avoid using absolute values for object locators. Please frame this in relative format